src/classes/CW_AssetTypeApprovalProcess.cls

```diff
@@ -0,0 +1,34 @@
+/**
+* File Name   :    CW_AssetTypeApprovalProcess.cls
+* Description :    B-108927 Approve / Reject Mailer Dispatch Dates
+*				   B-108925 Approve / Reject Teaser Dates
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 16/05/2014 Cristina P.	Intial
+*
+*/
+public with sharing class CW_AssetTypeApprovalProcess 
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.